tools/ci-build/README.md

```diff
@@ -0,0 +1,39 @@
+ci-build
```

Comment:
not sure how this ends up working, but I usually end up also needing a `.dockerignore` file to prevent unnecessary files being sent to the docker daemon when building. If docker builds stall for a few seconds when you start them locally, that may be why

Reply:
I haven't noticed any issues here. If it sends the directory the Dockerfile is from, then those files are trivially small in this case.